improving the early years page (#3073)
DFE-Digital/get-into-teaching-app: app/views/content/early-years-teaching-training.md

description: |-
  Discover how to get Early Years Teacher Status (EYTS) and help to educate and inspire future generations. Official guidance on early years teacher training.
calls_to_action:
  EY-graduate:
    name: feature_table
    arguments:

>       - Cost: "£7000 course fees are covered by the Department for Education and are paid directly to the training provider. You don’t need to do anything to access this funding."

Reviewer: ```suggestion
      - Cost: "£7000 course fees are covered by the Department for Education and are paid directly to the training provider. You do not need to do anything to access this funding."
```

        Salary: "No"
        Length of training: "Typically 1 academic year"
        Qualification: "Early years teacher status"
  EY-graduate-employment:
    name: feature_table
    arguments:
      - Cost: "£7000 course fees are covered by the Department for Education and are paid directly to the training provider. You don’t need to do anything to access this funding."
        Salary: "Provided by employer"
        Length of training: "Typically 1 academic year"
        Qualification: "Early years teacher status"
  EY-undergraduate:
    name: feature_table
    arguments:
      - Cost: "Up to £9,250"
        Salary: "No"
        Length of training: "3 or 4 years"
        Qualification: "Early years teacher status"
  EY-self-funded:
    name: feature_table
    arguments:

>       - Cost: "Self-funded"

Reviewer: Do we have the actual cost for this? seems a bit strange that we don't

Author: Good question, I'll check with the team.

Reviewer 2: It may vary by provider - if there is no exact figure, is there a 'Typically between x and y' statement we could add?

        Salary: "No"
        Length of training: "3 months"
        Qualification: "Early years teacher status"
---

The early years are critical in children’s development. As an early years practitioner you’ll work with young children up to the age of 5. You’ll use your knowledge and skills to help young children enjoy high standards of teaching and open their minds to new ideas every day.

You’ll need to do ‘early years initial teacher training’ (EYITT) to achieve ‘early years teacher status’ (EYTS). This will allow you to work in a nursery or early years setting. 

> If you want to teach in a school then you will need to achieve qualified teacher status (QTS). [Find out more about training to teach primary and secondary pupils](/train-to-be-a-teacher).

Reviewer: ```suggestion
If you want to teach in a primary or secondary school then you will need to achieve qualified teacher status (QTS). [Find out more about training to teach primary and secondary pupils](/train-to-be-a-teacher).
```

## What you need to become an early years practitioner

> To do postgraduate early years teacher training, you’ll need a bachelor’s degree or an equivalent including vocational, professional, and overseas qualifications.

Reviewer: Don't know if I'm just being a bit picky (probs!) but not sure if this makes sense to me like this, just because further down you talk about undergraduate training? I think if I read this first, I'd just assume all training is postgrad then and not read further? Not sure if this needs some sort of extra subheading/context, if that makes sense?

Author: It's a good point. I've added a line to clarify. Do you think this makes it clearer @EllieNodder?

> description: |-

Author: Personally, I don't think it's worth changing. The vast majority of traffic comes from search so I doubt most users even notice the URL.

> description: |-

Reviewer: could be worth asking the early years team? is this something they've noticed their users struggle with? (not suggesting we hold up this PR, but could consider after!)

  Discover how to get Early Years Teacher Status (EYTS) and help to educate and inspire future generations. Official guidance on early years teacher training.
calls_to_action:
  EY-graduate:
    name: feature_table
    arguments:

>       - Cost: "£7000 course fees are covered by the Department for Education and are paid directly to the training provider. You do not need to do anything to access this funding."

Reviewer 2: ```suggestion
      - Cost: "£7,000 course fees are covered by the Department for Education and are paid directly to the training provider. You do not need to do anything to access this funding."
```

Reviewer 2: If the fees are covered by DfE, do we need to tell candidates how much they are? (Maybe yes if we think this would help encourage people by showing how much they are 'saving' / being valued?)

If we think that we do need to specify the amount, should it be upfront like this? If I was scanning this as a user, is there a danger that I would focus on the £7k and miss the fact that DfE pays for it?

Author: Good point, I was thinking it would help encourage as you describe but maybe need to think again about the ordering.

        Salary: "No"
        Length of training: "Typically 1 academic year"
        Qualification: "Early years teacher status"
  EY-graduate-employment:
    name: feature_table
    arguments:

>       - Cost: "£7000 course fees are covered by the Department for Education and are paid directly to the training provider. You do not need to do anything to access this funding."

Reviewer 2: ```suggestion
      - Cost: "£7,000 course fees are covered by the Department for Education and are paid directly to the training provider. You do not need to do anything to access this funding."
```

Reviewer 2: As above re displaying the amount / displaying it as the lead!

        Salary: "Provided by employer"
        Length of training: "Typically 1 academic year"
        Qualification: "Early years teacher status"
  EY-undergraduate:
    name: feature_table
    arguments:
      - Cost: "Up to £9,250"
        Salary: "No"
        Length of training: "3 or 4 years"
        Qualification: "Early years teacher status"
  EY-self-funded:
    name: feature_table
    arguments:
      - Cost: "Self-funded"
        Salary: "No"
        Length of training: "3 months"
        Qualification: "Early years teacher status"
---

The early years are critical in children’s development. As an early years practitioner you’ll work with young children up to the age of 5. You’ll use your knowledge and skills to help young children enjoy high standards of teaching and open their minds to new ideas every day.

> You’ll need to do ‘early years initial teacher training’ (EYITT) to achieve ‘early years teacher status’ (EYTS). This will allow you to work in a nursery or early years setting. 

Author: I agree, removed.

If you want to teach in a primary or secondary school then you will need to achieve qualified teacher status (QTS). [Find out more about training to teach primary and secondary pupils](/train-to-be-a-teacher).

## What you need to become an early years practitioner

You can train to become an early years practioner as a postgraduate if you already have a degree, or as part of an undergraduate degree. 

> To do postgraduate training, you’ll need a bachelor’s degree or an equivalent including vocational, professional, and overseas qualifications.

Reviewer 2: ```suggestion
To do postgraduate training, you’ll need a bachelor’s degree or an equivalent qualification including vocational, professional, and overseas qualifications.
```
Is 'including' the right word here? I'm struggling to suggest something better though!

Author: Tweaked it to 'for example'

To start early years teacher training, you’ll need these qualifications (or to demonstrate an equivalent standard):

> * maths grade 4 (C)

Reviewer 2: don't we need to specify that it is maths **GSCE** grade 4 (C)?

NB I know it is like this in production but it seems very odd not to say what qualification we are talking about?

  EY-undergraduate:
    name: feature_table
    arguments:
      - Cost: "Up to £9,250"
        Salary: "No"
        Length of training: "3 or 4 years"
        Qualification: "Early years teacher status"
  EY-self-funded:
    name: feature_table
    arguments:
      - Cost: "This could be around £2,500 or £3,000 depending on the provider"
        Salary: "No"
        Length of training: "3 months"
        Qualification: "Early years teacher status"
---

The early years are critical in children’s development. As an early years practitioner you’ll work with young children up to the age of 5. You’ll use your knowledge and skills to help young children enjoy high standards of teaching and open their minds to new ideas every day.

You’ll need to do early years initial teacher training (EYITT) to achieve early years teacher status (EYTS). This will allow you to work in a nursery or early years setting. 

If you want to teach in a primary or secondary school then you will need to achieve qualified teacher status (QTS). [Find out more about training to teach primary and secondary pupils](/train-to-be-a-teacher).

## What you need to become an early years practitioner

> There are 2 ways to train to become an early years practioner, either as a postgraduate if you already have a degree, or as part of an undergraduate degree. 

Reviewer 2: ```suggestion
There are 2 ways to train to become an early years practitioner, either as a postgraduate if you already have a degree, or as part of an undergraduate degree. 
```

You’ll need to do early years initial teacher training (EYITT) to achieve early years teacher status (EYTS). This will allow you to work in a nursery or early years setting. 

If you want to teach in a primary school, including in a reception class, or secondary school then you will need to achieve qualified teacher status (QTS). [Find out more about training to teach primary and secondary pupils](/train-to-be-a-teacher).

## What you need to become an early years teacher

There are 2 ways to train to become an early years teacher, either as a postgraduate if you already have a degree, or as part of an undergraduate degree. 

To do postgraduate training, you’ll need a bachelor’s degree or an equivalent, for example, a vocational, professional, or overseas qualification.

To start early years teacher training, you’ll need these GCSE qualifications (or a vocational, professional, or overseas qualification to the same standard):

* maths grade 4 (C)
* English grade 4 (C)
* science grade 4 (C)

> ### Non-UK candidates

Reviewer 2: Thanks for adding this in @philspray !

Rather than having it as an h3, should we be using our inset component - wasn't this the way we were going to flag different info for non-uk candidates across the site? Paging @EllieNodder as I might be wrong!

Reviewer: yeah I think the inset component here makes sense

> You can can apply for early years initial teacher training, if you meet all [the academic entry criteria and the relevant immigration permissions](https://www.gov.uk/guidance/early-years-initial-teacher-training-2022-to-2023-funding-guidance#international-students).

Reviewer 2: ```suggestion
You can can apply for early years initial teacher training, if you meet all the [academic entry criteria and immigration permissions for non-UK candidates](https://www.gov.uk/guidance/early-years-initial-teacher-training-2022-to-2023-funding-guidance#international-students).
```

Does this link text need to mention non-UK candidates explicitly to give more context?

> You can can apply for early years initial teacher training, if you meet all the [academic entry criteria and immigration permissions for non-UK candidates](https://www.gov.uk/guidance/early-years-initial-teacher-training-2022-to-2023-funding-guidance#international-students).

Reviewer: ```suggestion
You can apply for early years initial teacher training, if you meet all the [academic entry criteria and immigration permissions for non-UK candidates](https://www.gov.uk/guidance/early-years-initial-teacher-training-2022-to-2023-funding-guidance#international-students).
```

## Routes to early years teacher status

### Postgraduate training
This route is designed for people who have an undergraduate degree and want to continue full-time study to achieve early years teacher status. 

$EY-graduate$

Bursaries are only available for this route as follows:

* £5,000 if you have a first class degree
* £4,000 if you have a 2:1 degree
* £2,000 if you have a 2:2 degree

You are not eligible for a bursary if you do not have these qualifications or if you're completing any other EYITT  routes.

You do not need to do anything to access this funding. You can [learn more about early years initial teacher training funding](https://www.gov.uk/guidance/early-years-initial-teacher-training-2023-to-2024-funding-guidance).

### Postgraduate employment-based training
This route is for graduates working in an early years setting who want to achieve early years teacher status.    

$EY-graduate-employment$

### Undergraduate training

> This route is for people who want to study a full-time bachelor’s degree in an early-childhood related subject leading to EYTS.

Reviewer 2: ```suggestion
This route is for people who want to study for a full-time bachelor’s degree in an early-childhood related subject leading to EYTS.
```
Does this need `for`?

* £4,000 if you have a 2:1 degree
* £2,000 if you have a 2:2 degree

> You aren’t eligible for a bursary if you have a third class degree.

Reviewer: ```suggestion
You are not eligible for a bursary if you have a third class degree.
```

> You don’t need to do anything to access this funding. You can [learn more about early years initial teacher training funding](https://www.gov.uk/guidance/early-years-initial-teacher-training-2023-to-2024-funding-guidance).

Reviewer: ```suggestion
You do not need to do anything to access this funding. You can [learn more about early years initial teacher training funding](https://www.gov.uk/guidance/early-years-initial-teacher-training-2023-to-2024-funding-guidance).
```

### Postgraduate employment-based training
This is a route for graduates working in an early years setting who want to achieve early years teacher status.    

$EY-graduate-employment$

Up to £14,000 of funding is available. This would cover course fees of up to £7,000, plus £7,000 for your employer, to help cover their extra costs while you train. 

> You don’t need to do anything to access this funding. You can [learn more about early years inital teacher training funding](https://www.gov.uk/guidance/early-years-initial-teacher-training-2023-to-2024-funding-guidance).

Reviewer: ```suggestion
You do not need to do anything to access this funding. You can [learn more about early years inital teacher training funding](https://www.gov.uk/guidance/early-years-initial-teacher-training-2023-to-2024-funding-guidance).
```

> You don’t need to do anything to access this funding. You can [learn more about early years inital teacher training funding](https://www.gov.uk/guidance/early-years-initial-teacher-training-2023-to-2024-funding-guidance).

Reviewer: Do we need to have this link twice do you think?

* £4,000 if you have a 2:1 degree
* £2,000 if you have a 2:2 degree

You are not eligible for a bursary if you have a third class degree.

You do not need to do anything to access this funding. You can [learn more about early years initial teacher training funding](https://www.gov.uk/guidance/early-years-initial-teacher-training-2023-to-2024-funding-guidance).

### Postgraduate employment-based training

> This is a route for graduates working in an early years setting who want to achieve early years teacher status.    

Reviewer 2: ```suggestion
This route is for graduates working in an early years setting who want to achieve early years teacher status.    
```
consistent with other sections?

$EY-graduate-employment$

> Up to £14,000 of funding is available. This would cover course fees of up to £7,000, plus £7,000 for your employer, to help cover their extra costs while you train. 

Reviewer 2: Does this sound slightly odd? The funding isn't available to candidates (if I'm reading it right) so it isn't it just about highlighting the additional (up to) £7k for your employer?

(NB I appreciate that it says this in prod at the moment!)

You do not need to do anything to access this funding. You can [learn more about early years inital teacher training funding](https://www.gov.uk/guidance/early-years-initial-teacher-training-2023-to-2024-funding-guidance).

### Undergraduate training
This route is for people who want to study a full-time bachelor’s degree in an early-childhood related subject leading to EYTS.

$EY-undergraduate$

Undergraduate tuition fee loans are available from [Student Finance England (SFE)](https://www.gov.uk/student-finance).

> ### Postgraduate 'assessment only' training

Reviewer 2: do we need quotes around assessment only? (I appreciate they are in prod at the moment!)

> This self-funded route is ideal for graduates with early-years experience. If you already meet the [early years teachers’ standards](https://www.gov.uk/government/publications/early-years-teachers-standards) you can gain EYTS without the need for further training.

Reviewer 2: Does 'early-years' need a hyphen here? This is the only place on the page (but I don't know if that is because of the grammatical necessity in this context!)

$EY-self-funded$

> ## Find your early years teacher training course

Reviewer 2: Is this h2 more about 'How to apply for early years teacher training'?

You should contact accredited early years training providers directly to apply for a place.

> <a href="https://www.gov.uk/government/publications/early-years-initial-teacher-training-itt-providers-and-school-direct-early-years-lead-organisations/early-years-initial-teacher-training-itt-providers-and-school-direct-lead-organisations" class="button">See early years training providers</a>

Reviewer 2: ```suggestion
<a href="https://www.gov.uk/government/publications/early-years-initial-teacher-training-itt-providers-and-school-direct-early-years-lead-organisations/early-years-initial-teacher-training-itt-providers-and-school-direct-lead-organisations" class="button">Find an early years training provider</a>
```
Find or Choose rather than See?
